src/test/resources/mockdata/invoices/f847a34e-0df2-40b9-bc84-cfa880ef3c8b.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Instead of adding new mock file, you can use `MockServer.addMockEntry` and using, for example, `ApiTestBase.getMinimalContentInvoice()` adding required data...

Reply:
Done